addons/account/tests/test_chart_template.py

```diff
@@ -663,7 +663,7 @@ def local_get_data(self, template_code):
         with patch.object(AccountChartTemplate, '_get_chart_template_mapping', side_effect=local_get_mapping, autospec=True):
             with patch.object(AccountChartTemplate, '_get_chart_template_data', side_effect=local_get_data, autospec=True):
                 with patch.object(AccountChartTemplate, '_post_load_data', wraps=test_post_load_data):
-                    with patch.object(code_translations, 'python_translations', mock_python_translations):
+                    with patch.object(code_translations, 'get_python_translations', mock_python_translations):
```

Comment:
Hello, thanks for the PR. Why do you do this change ? Your commit message does not explain why you change the dictionary object `python_translations` to the function `get_python_translations`. This is wrong since `mock_python_translations` is itself a dictionary.  The error you get with the upgrade seems only related to the wrong import, not this test. With your change, the test `TestChartTemplate.test_install_with_translations` is failing